homeassistant/components/asuswrt/__init__.py

```diff
@@ -77,10 +78,16 @@
         dnsmasq=conf[CONF_DNSMASQ],
     )
 
-    await api.connection.async_connect()
-    if not api.is_connected:
-        _LOGGER.error("Unable to setup component")
-        return False
+    try:
+        await api.connection.async_connect()
+        if not api.is_connected:
+            _LOGGER.error("Unable to setup component")
+            return False
+    except OSError as ex:
+        """Raise platform not ready to retry setup on network startup."""
+        raise PlatformNotReady() from ex
+    except:
+        raise
```

Comment:
If you don't want to catch it, simply don't.
```suggestion
```

Reply:
Corrected as suggested